### ISSUE TITLE
New CST 2.0 autoscale example userData.yaml has bad environment for azure

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-azure-arm-templates#208]
## Do you already have an issue opened with F5 support?
No

## Description
Environment example in the userData.yaml file for the autoscale folder has environment listed as "azure,,,". This fails. It needs to be only "azure".

Current:
```
runtime_parameters:
  - metadataProvider:
      environment: azure¸¸¸
      field: name
      type: compute
    name: HOST_NAME
    type: metadata
```

Should be...
```
runtime_parameters:
  - metadataProvider:
      environment: azure
      field: name
      type: compute
    name: HOST_NAME
    type: metadata
```

Error when bad environment:
```
2020-09-20T20:55:04.499Z [19688]: info: Configuration file: /config/cloud/onboard_config.yaml
2020-09-20T20:55:04.511Z [19688]: info: Validating provided declaration
2020-09-20T20:55:04.654Z [19688]: info: Invalid declaration: "data.runtime_parameters[0].metadataProvider.environment should be equal to one of the allowed values"
2020-09-20T20:55:04.655Z [19688]: info: Sending F5 Teem report for failure case.
2020-09-20T20:55:16.513Z [19688]: info: {"id":"0b868593-57a7-ee45-fae0ee2a71ad","product":"BIG-IP","cpuCount":4,"diskSize":108544,"memoryInMb":14033,"version":"15.1.0.4","nicCount":2,"regKey":"KVKMY-TGVEZ-FXDLU-ZBKMY-FMAYERM","platformId":"Z100","hostname":"bigip1","management":"10.0.0.4/24","provisionedModules":{"ltm":"nominal"},"installedPackages":{},"environment":{"pythonVersion":"Python 2.7.5","pythonVersionDetailed":"2.7.5 (default, Jun 18 2020, 19:39:50) \n[GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]","nodeVersion":"v6.9.1","libraries":{"ssh":"OpenSSH_7.4p1, OpenSSL 1.0.2s-fips  28 May 2019"}}}
2020-09-20T20:55:16.520Z [19688]: warn: AWS did not work. Trying another cloud
2020-09-20T20:55:16.520Z [19688]: error: Cannot find module 'aws-sdk'
2020-09-20T20:55:26.980Z [19688]: info: F5 Teem report was successfully sent for failure case.
2020-09-20T20:55:26.980Z [19688]: error: Invalid declaration: "data.runtime_parameters[0].metadataProvider.environment should be equal to one of the allowed values"
```

## Template
https://github.com/F5Networks/f5-azure-arm-templates/blob/master/examples/autoscale/userData.yaml

## Severity Level
4

**Comment by shyawnkarim** [comment]
Thanks for reporting this issue. I've filed a bug, internal ID ESECLDTPLT-2226, to get this fixed.

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-azure-arm-templates-v2).
